### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by poulainv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,150 @@
...
+
+## Product
+
+First goal: designing a product human centered allowing people to build and manage their online and public library. Just a tool to manage and gather the content we love, in order to share it with friends & community 😇
+
+### Why ?
+
+Social medial are now all stream based. More and more, humans — especially journalists, are loosing their ability to choose which content to promote and amplify. Instead, recommendation systems feed a unique & personnal narrative stream of content, that only consider clicks and views. Of course, what else they can do? [Here, I'm happy to share some references](https://beta.tottem.app/vincent/c/inspirational-content-about-why-social-media-companies-are-dangerous-for-personal-attention-and-democracy-ck5i4lwp2000vws9e4ry25feh)
+
+So, what if I want to explore a durable friend's books or articles recommendation? What if I want to really dig a specific subject?
```
Comment (deonclem) [inline README.md]: So, what if I want to explore durable book or article recommendations from a friend? What if I want to really dig into a specific subject?

File: README.md [hunk 2]
```diff
@@ -1 +1,150 @@
...
+-   building a product based on ethic design
+-   experimenting open-source web technologies and share it
+
+## Product
+
+First goal: designing a product human centered allowing people to build and manage their online and public library. Just a tool to manage and gather the content we love, in order to share it with friends & community 😇
+
+### Why ?
+
+Social medial are now all stream based. More and more, humans — especially journalists, are loosing their ability to choose which content to promote and amplify. Instead, recommendation systems feed a unique & personnal narrative stream of content, that only consider clicks and views. Of course, what else they can do? [Here, I'm happy to share some references](https://beta.tottem.app/vincent/c/inspirational-content-about-why-social-media-companies-are-dangerous-for-personal-attention-and-democracy-ck5i4lwp2000vws9e4ry25feh)
```
Comment (deonclem) [inline README.md]: Every social medial is now completely stream based. More and more, internet users (up to you 😄 ) — especially journalists, are loosing their ability to choose which content to promote and share(?). Instead, the current system, based on recommendations, will feed a unique & impersonal (did you actually mean personal 🤔 ?) narrative stream of content, that only cares about clicks and views. Of course, how else could it work? [Here, I'm happy to share some references](https://beta.tottem.app/vincent/c/inspirational-content-about-why-social-media-companies-are-dangerous-for-personal-attention-and-democracy-ck5i4lwp2000vws9e4ry25feh)

Not sure I correctly followed your train of thoughts here, I might have missed your point.

Comment (poulainv) [reply README.md]: Yes, by some manner. I combined

File: README.md [hunk 3]
```diff
@@ -1 +1,150 @@
-# quiet
\ No newline at end of file
+Tottem is an open source experimentation, it tries to combine personal productivity tool approach with (slow) social media capabilities to make users empowered and somehow emancipated.
```
[the remark this answers is not in the document]
Comment (poulainv) [reply README.md]: I want to keep "emcipated" :)

**Comment by poulainv** [comment]
THANKS!

**Pull request merged**